colossalai/utils/model/lazy_init_context.py

```diff
@@ -8,6 +8,13 @@
 import typing
 from typing import List, Callable
 
+
+def _substitute_init_recursively(cls, func):
+    for subcls in cls.__subclasses__():
+        _substitute_init_recursively(subcls, func)
+        func(subcls)
```

Comment:
Can we import this method from the ctx utils directly?

Reply:
Done